docs/server/security/user-authentication.md

```diff
@@ -506,9 +506,9 @@ You need to configure the following:
 			"AllowedScopes": [
 				"streams",
 				"openid",
-				"profile",
+				"profile"
 			],
-			"RedirectUris": ["https://localhost:2113/oauth/callback","https://127.0.0.1:2113/oauth/callback"],
+          "RedirectUris": ["https://localhost:2113/signin-oidc","https://127.0.0.1:2113/signin-oidc","https://localhost:2113/oauth/callback","https://127.0.0.1:2113/oauth/callback"],
```

Comment:
Please make sure the indentation is correct
<img width="675" alt="image" src="https://github.com/user-attachments/assets/a7077a6b-47f5-4de0-962a-a34dde35600a" />